SCClassLibrary/Common/GUI/PlusGUI/Control/server-meter.sc

```diff
@@ -1,10 +1,11 @@
 + Server {
-	meter { |numIns, numOuts|
-		if(numIns.notNil or: numOuts.notNil and: { serverMeter.notNil }) { serverMeter.close };
+	meter { |numIns, numOuts, position|
 		if(serverMeter.isNil or: { serverMeter.isClosed }) {
 			serverMeter = ServerMeter(this, numIns, numOuts)
 		} {
 			serverMeter.window.front
-		}
+		};
+		position !? {serverMeter.position_(position) };
```

Comment:
add a blank space after { 
(sorry!)

Reply:
No problem, thanks!
Done in https://github.com/supercollider/supercollider/pull/6283/commits/f4b583e070b7af46a4346390abae65532bc7ea40